pkg/providers/vsphere/vsphere_test.go

```diff
@@ -533,6 +533,12 @@ func TestProviderGenerateCAPISpecForUpgradeUpdateMachineTemplate(t *testing.T) {
 			wantCPFile:        "testdata/expected_results_minimal_cp.yaml",
 			wantMDFile:        "testdata/expected_results_minimal_md.yaml",
 		},
+		{
```

Comment:
Should I make my own test for this? it fits nicely in this one, but I wasn't sure.